tests/contrib/httplib/test_httplib.py

```diff
@@ -242,7 +241,8 @@ def test_httplib_request_get_request_query_string(self):
             {
                 'http.method': 'GET',
                 'http.status_code': '200',
-                'http.url': '{}?key=value&key2=value2'.format(URL_200),
+                # check url metadata lacks query string
```

Comment:
oh, sorry, it is L223 in this file that needs to change:

`we capture the entire url in the span`

which is not longer true.